mmv1/products/certificatemanager/terraform.yaml

```
@@ -44,15 +44,30 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           cert_name: "dns-cert"
         ignore_read_extra:
           - "managed.0.dns_authorizations"
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "certificate_manager_self_managed_certificate"
+        primary_resource_id: "default"
+        vars:
+          cert_name: "self-managed-cert"
+        ignore_read_extra:
+          - "self_managed.pem_certificate"
```

You'll also need the `0` index in here as well, and for `self_managed.0.pem_private_key` below
```suggestion
          - "self_managed.0.pem_certificate"
```